kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -10,19 +10,23 @@
 
 
 class ScatteringBase1D(ScatteringBase):
-    def __init__(self, J, shape, Q=1, T=None, max_order=2, average=True,
-            oversampling=0, out_type='array', backend=None):
+    def __init__(self, J, shape, Q=1, T=None, max_order=2, average=None, 
+                 oversampling=0, out_type='array', backend=None):
         super(ScatteringBase1D, self).__init__()
         self.J = J
         self.shape = shape
         self.Q = Q
         self.T = T
         self.max_order = max_order
-        self.average = average
         self.oversampling = oversampling
         self.out_type = out_type
         self.backend = backend
 
+        if average is not None:
+            warn("`average` is deprecated v0.3 and will be removed in v0.4."
+                 "Replace `average=False` by `T=0` and set `T>1` or leave" 
+                 "`T=None` for `average=True` (default)", DeprecationWarning)
```

Comment:
"Replace `average=False` by `T=0`; for `average=True` , either use the default averaging by `T=None` or customize averaging by setting T to a positive integer" => something like this to make it clearer?